client/python/README.md

```diff
@@ -0,0 +1,181 @@
+
+init the project:
+```bash
+$ uv init
+# Initialized project `python-test`
+```
+
+add the SDK to your project:
+```bash
+$ uv add git+https://github.com/agntcy/dir.git@main#subdirectory=client/python
+# Using CPython 3.12.7
+# Creating virtual environment at: .venv
+#     Updated https://github.com/agntcy/dir.git (ec4c0b36e7d483ac19341795398c280e7298e383)
+# Resolved 12 packages in 887ms
+#       Built agntcy-dir-client-sdk @ git+https://github.com/agntcy/dir.git@ec4c0b36e7d483ac19341795398c280e7298e383#subdirectory=client/python
+#       Built agntcy-dir-proto-stubs @ git+https://github.com/agntcy/dir.git@ec4c0b36e7d483ac19341795398c280e7298e383#subdirectory=api/python
+# Prepared 4 packages in 1.31s
+# Installed 10 packages in 18ms
+#  + agntcy-dir-client-sdk==0.1.0 (from git+https://github.com/agntcy/dir.git@ec4c0b36e7d483ac19341795398c280e7298e383#subdirectory=client/python)
+#  + agntcy-dir-proto-stubs==0.1.0 (from git+https://github.com/agntcy/dir.git@ec4c0b36e7d483ac19341795398c280e7298e383#subdirectory=api/python)
+#  + grpcio==1.71.0
+#  + grpcio-tools==1.71.0
+#  + iniconfig==2.1.0
+#  + packaging==25.0
+#  + pluggy==1.5.0
+#  + protobuf==5.29.4
+#  + pytest==8.3.5
+#  + setuptools==80.3.1
+```
+
+## Usage
+
+### Starting the AGNTCY Server
+
+For these commands to work correctly, you need to have the **AGNTCY** server running and accessible.
+
+```bash
+$ task server start
+# task: [server:store:start] # mount config
+# cat > /tmp/config.json <<EOF
+# {
+#   "distSpecVersion": "1.1.1",
+#   "storage": {
+#     "rootDirectory": "/tmp/zot"
+#   },
+#   "http": {
+#     "address": "127.0.0.1",
+#     "port": "5000"
+#   },
+#   "log": {
+#     "level": "debug"
+#   },
+#   "extensions": {
+#     "search": {
+#       "enable": true,
+#       "cve": {
+#         "updateInterval": "24h"
+#       }
+#     }
+#   }
+# }
+# EOF
+# 
+# # run docker with attached volume
+# docker run \
+#       -it \
+#       --rm -d -p 5000:5000 \
+#       -v /tmp/config.json:/config.json:ro \
+#       --name oci-registry ghcr.io/project-zot/zot-linux-arm64:v2.1.1
+# 
+# 1f772921467d2eaa6070ddb857500f29ace22aeea75cd75d06c67a157a0a00e1
+# task: [server:start] go run main.go
+# time=2025-05-08T12:35:48.942+02:00 level=INFO msg="Config file not found, use defaults." component=config
+# time=2025-05-08T12:35:48.952+02:00 level=INFO msg="Ignoring announcement event for invalid object" component=routing/handler digest="sha256:̪뉍\r\xd2\xd4X\xaa\x14\xe7\xcc=\xd4y\xbb>\xd0\xc9\x04\xf8\x99]]\xaf\xc7\xfa\xdev0J"
+# INFO[0000] Starting healthz server. listenAddr=0.0.0.0:8889
+# time=2025-05-08T12:35:48.952+02:00 level=INFO msg="Server starting" component=server address=0.0.0.0:8888
+# time=2025-05-08T12:37:48.951+02:00 level=INFO msg="Ignoring announcement event for invalid object" component=routing/handler digest="sha256:̪뉍\r\xd2\xd4X\xaa\x14\xe7\xcc=\xd4y\xbb>\xd0\xc9\x04\xf8\x99]]\xaf\xc7\xfa\xdev0J"
+```
+
+### Example Usage
+
+```python
```

Comment:
i would suggest to break these things into multiple parts based on your comments, like "Create an agent object", "Push", "Pull", etc

I would also convert this example into an additional file that can be used directly, like:
`example.py`